shaders/glsl/renderchunk.fragment

```diff
@@ -496,7 +475,7 @@ void main()
 	fresnel = mix(fresnel, 1.0, farWaterFresnel * isWater);
 	
 
-	//the same tweak with puddles
+	// The same fix but with puddles
```

Comment:
It is not a fix. So if it is not a tweak than it is a trick I think

Reply:
Ok I'll correct that